doc/sphinxext/numpy_ext/docscrape_sphinx.py

```diff
@@ -27,7 +27,6 @@ def _str_indent(self, doc, indent=4):
         return out
 
     def _str_signature(self):
-        return ['']
```

Comment:
git blame give me :
commit 62ac02846bea84e5856b1742afcd4f650d2a8440
Date:   Mon Sep 6 09:41:49 2010 +0200
ENH: Make sure docs build on older versions of sphinx

I think that after 6 years we can drop some old versions of sphinx.